src/Components/Publishing/Fixtures/Articles.ts

```diff
@@ -685,19 +685,33 @@ export const VideoArticle = {
   published_at: "2017-07-28T20:38:05.709Z",
   description: "The elegant spiral of the Nautilus shell, the sinuous pattern of the banks of a river, or the swirling vortex street of clouds - patterns exist.",
   media: {
-    url: "https://artsy-media-uploads.s3.amazonaws.com/z9w_n6UxxoZ_u1lzt4vwrw%2FHero+Loop+02.mp4",
+    title: "Trevor Paglan",
```

Comment:
Looks like this info is repeated below and in the components file, maybe you could import this data from there?